cytominer_eval/operations/percent_strong.py

```diff
@@ -38,13 +39,17 @@ def percent_strong(
     # Check to make sure that the melted dataframe is upper triangle
     assert_melt(similarity_melted_df, eval_metric="percent_strong")
 
+    # check that there ARE group_replicates (non-unique rows)
```

Comment:
```suggestion
    # check that there are group_replicates (non-unique rows)
```

super minor, but we can drop the emphasis